.NET/Microsoft.Recognizers.Text.DateTime/English/Parsers/EnglishDatePeriodParserConfiguration.cs

```diff
@@ -8,6 +8,8 @@ namespace Microsoft.Recognizers.Text.DateTime.English
 {
     public class EnglishDatePeriodParserConfiguration : BaseOptionsConfiguration, IDatePeriodParserConfiguration
     {
+        private const string AfterNextSuffix = "after next";
```

Comment:
Don't add as string, if needed, add as resource in YAML. Even better if it can support multiple values.